src/resources/Indexes/IndexesInterface.ts

```diff
@@ -3,6 +3,7 @@ export interface IndexStorageSpec {
     sizeInGibibytes: number;
     sizeInGigabytes: number;
     storageType: string;
+    throughputInMebibytes: number;
```

Comment:
Would it be a good opportunity to add some JsDoc?

Reply:
I wont unless I am obligated. My knowledge of the index processes is near to zero, so I would clearly not be the best person to add any doc on those calls and interfaces.